replica/table.cc

```diff
@@ -2010,7 +2011,7 @@ locator::table_load_stats table::table_load_stats(std::function<bool(locator::gl
     stats.split_ready_seq_number = _sg_manager->split_ready_seq_number();
 
     for (unsigned id = 0; id < storage_groups().size(); id++) {
-        auto& sg = storage_groups()[id];
+        auto& sg = storage_groups().at(id);
```

Comment:
ditto, `at(id)` would throw.
wouldn't `for (const auto& [id, sg] : storage_groups())` work better here?
and then the (!sg) below would probably need to be an internal error after dynamic allocation of storage groups is implemented.

Reply:
yes, it should have been "for (const auto& [id, sg] : storage_groups())", thinking forward to dynamic allocation. thanks for catching it!